File: 1768-merge-strings-alternately/1768-merge-strings-alternately.java
import java.util.Scanner;
import java.lang.StringBuilder;
class Solution {
    public String mergeAlternately(String word1, String word2) {
        int length = (word1.length() >= word2.length())
            ? word1.length()
            : word2.length()
        ;
        String merged = "";
        StringBuilder sb = new StringBuilder(merged);
        for (int i = 0; i < length; i++){
            if (i < word1.length()){
                sb.append(word1.charAt(i));
            }
            if (i < word2.length()){
                sb.append(word2.charAt(i));
            }
        }


        return sb.toString();

    }
}

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cs
public class Solution {
    public string MergeAlternately(string word1, string word2) {
        // while (String.indexOf())
        int i = 0;
        string word3 = "";
        while (i < word1.Length && i < word2.Length){
            word3 +=word1[i].ToString() + word2[i++].ToString();
        }
        return word3 + ((word1.Length > word2.Length) ? word1.Substring(i) : word2.Substring(i));
    }
}